server/src/main/java/kr/codesquad/library/controller/BookSearchController.java

```diff
@@ -40,4 +40,11 @@
         return ResponseEntity.ok(OK(bookSearchService.findByCategory(categoryId, page)));
     }
 
+    @ApiOperation(value = "도서 상세페이지")
+    @GetMapping("/books/{bookId}")
+    public ResponseEntity<ApiResult> getBookDetail(@PathVariable Long bookId) {
```

Comment:
`@RestController` 를 사용했다면 ResponseEntity 는 생략되어도 되지 않나요?
```suggestion
    public ApiResult getBookDetail(@PathVariable Long bookId) {
```